hw/ip/aes/data/aes_testplan.hjson

```diff
@@ -0,0 +1,71 @@
+/ Copyright lowRISC contributors.
```

Comment:
There's a second `/` missing, that's why CI and doc builds are failing.

Reply:
ahh thank you I was scratching my head..